### PR TITLE
Vendor update go-iptables

### PR DESCRIPTION
**Pull request by mccv1r0**
Vendor update go-iptables to obtain commit f1d0510cabcb710d5c5dd284096f81444b9d8d10
Update go.mod & go.sub

### CODE REVIEW

**Comment by mars1024** [comment]
Otherwise, I think we need a stable version, like 0.4.2, maybe @squeed can help to release one.

**Comment by mccv1r0** [comment]
> Could you help to describe that why we need this vendor-update? Some bug fix or functional upgrade? 

There was a work-around in the iptables code for nftables which is no longer needed, and now breaks things when nftables is used.

**Comment by mars1024** [comment]
> > Could you help to describe that why we need this vendor-update? Some bug fix or functional upgrade?
> 
> There was a work-around in the iptables code for nftables which is no longer needed, and now breaks things when nftables is used.

Thanks!

**Comment by squeed** [comment]
Oh, silly me, I thought we'd released those features already. I'll do a go-iptables release now.

**Comment by squeed** [comment]
OK, tagged 0.4.2.

**Comment by mars1024** [comment]
> Oh, silly me, I thought we'd released those features already. I'll do a go-iptables release now.

Haha :), not too late, thanks! @squeed

File: go.sum [hunk 1]
```diff
@@ -12,6 +12,10 @@ github.com/coreos/go-iptables v0.3.0 h1:UTQkjHl9rPwwtXZhXbY3T932cV9aUnKlSsZ7YGfJ
 github.com/coreos/go-iptables v0.3.0/go.mod h1:/mVI274lEDI2ns62jHCDnCyBF9Iwsmekav8Dbxlm1MU=
 github.com/coreos/go-iptables v0.4.1 h1:TyEMaK2xD/EcB0385QcvX/OvI2XI7s4SJEI2EhZFfEU=
 github.com/coreos/go-iptables v0.4.1/go.mod h1:/mVI274lEDI2ns62jHCDnCyBF9Iwsmekav8Dbxlm1MU=
+github.com/coreos/go-iptables v0.4.2-0.20190627140312-f1d0510cabcb h1:9LWrtjSwtaPzlx8rZ5c5OGhJVMyJh7YhfFYypzwCAfM=
```
Comment (mars1024) [inline go.sum]: I think we need a **"go mod tidy"** to remove these useless lines.

Comment (mccv1r0) [reply go.sum]: agreed - done

**Comment by dcbw** [comment]
thanks, /lgtm

**Pull request merged**